addons/website/migrations/12.0.1.0/pre-migration.py

```diff
@@ -250,10 +250,10 @@ def migrate(env, version):
     cr = env.cr
     openupgrade.rename_fields(env, _field_renames)
     openupgrade.rename_xmlids(cr, _xml_ids_renames)
-    noupdate_changes(env)
     fill_website_company_id(cr)
     fill_website_name(cr)
     fill_website_redirect_type(cr)
     fill_website_redirect_urls(cr)
     disable_less_customizations(env)
     bootstrap_4_migration(env)
+    noupdate_changes(env)
```

Comment:
This was before the BS4 migration to avoid migrating pages that were going to get updated anyways. Please let it where it was before.

The real fix you're looking for is to [remove `homepage` from here](https://github.com/OCA/OpenUpgrade/pull/1923/files#diff-7f69b365074a905b3170de8b9b760af1R55).